lib/mercury/received_message.rb

```diff
@@ -38,14 +38,12 @@ def nack
     private
 
     def is_ackable
-      @work_queue_name != nil
+      !@work_queue_name.nil?
     end
 
     def performing_action(action)
-      is_ackable or raise "This message is not #{action}able"
-      if @action_taken
-        raise "This message was already #{@action_taken}ed"
-      end
+      is_ackable || raise("This message is not #{action}able")
+      raise "This message was already #{@action_taken}ed" if @action_taken
```

Comment:
I am generally fine with either approach - for instance in a class that is designed to verify some structure I'll often have a chain of lines:
`raise :foo unless foo`
`raise :bar unless bar`
`raise :foo_stupid if foo.stupid?`

Otherwise, I'll use this technique just to avoid the clutter of nesting.